PharmacyInfo/src/PharmaciesList.vue

```diff
@@ -75,6 +75,35 @@
                     </td>
                 </tr>
             </tbody>
+            <tbody v-else-if="authRole === 'Patient'">
+                <tr v-for="pharmacy in getPharmacies.filter(pharmacy=>pharmacy.Patients.filter(patient=>patient.PatientId === authId).length > 0)">
```

Comment:
Avoid such implementations in templates, there is should be only simple data usage, data and event bindings;
Move this code to the 'computed' fields, so then Vue will cache it